### PR TITLE
Adding multipul pices of functionality to Class Analytics

### PR DESCRIPTION
**Pull request by MichaelDillie**

### CODE REVIEW

**Comment by m2mathew** [comment]
:+1: 
A much needed addition to the arsenal

**Comment by whitneyklingensmith** [comment]
:+1:

File: scripts/components/ClassAnalyticsComponent.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+			);
+		});
+
+// {quiz.get('quizTitle')} - {quiz.get('totalQuestions')}
+
+		var rightContent = ('paul revere');
+
+
+		return (
+			<div className="class-anaylitcs-container">
```
Comment (Jmingus) [inline scripts/components/ClassAnalyticsComponent.js:131]: css classname is spelled wrong, not an error really, but you should be aware

**Comment by Jmingus** [comment]
besides the one warning, Everything looks good. :+1:

**Pull request merged**